Cargo.toml

```diff
@@ -7,4 +7,13 @@ members = [
 resolver = "2"
 
 [workspace.dependencies]
-alloy-sol-types = "0.7.7"
\ No newline at end of file
+sp1-sdk = "3.0.0"
+sp1-zkvm = "3.0.0"
+sp1-helper = "3.0.0"
+serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
+serde = { version = "1.0", default-features = false, features = ["derive"] }
+clap = { version = "4.0", features = ["derive", "env"] }
+tracing = "0.1.40"
+hex = "0.4.3"
+
+fibonacci-lib = { path = "lib" }
```

Comment:
Please set up your editor to end all lines, including the last line, with a newline.

[It's the POSIX standard](https://stackoverflow.com/a/729795/590534) and also makes GitHub and git complain less (and help with command line utilities like cat or grep, too).